tests/operators/test_datetime.py

```diff
@@ -52,30 +51,31 @@ def setup_class(cls):
         (datetime.time(10, 0, 0), datetime.datetime(2020, 7, 7, 11, 0, 0)),
     ]
 
-    def setup_method(self):
-        self.dag = DAG(
+    @pytest.fixture(autouse=True)
+    def base_tests_setup(self, dag_maker):
+        self.dag_maker = dag_maker
```

Comment:
I believe we don’t need to do this. Pytest reuses the same fixture objects during phases, so `dag_maker` in the argument list of a test function is automatically the same as the `dag_maker` in the fixture.